src/Config/upiconfig.xml

```diff
@@ -14,7 +14,7 @@
             <upi_attribute name='id' value='Win64'/>
             <upi_element name='level'>
               <upi_attribute name='name' value='build'/>
-              <upi_attribute name='id' value='2.15.0' />
+              <upi_attribute name='id' value='2.15.1' />
```

Comment:
For UPI, it will be under the same release but a different build, similar to how 2.13.1 is set up